Makefile

```diff
@@ -12,6 +12,7 @@ OUT_DIR = _output
 
 export GOFLAGS
 export TESTFLAGS
+export GO111MODULE=off
```

Comment:
What's the reason for this?  Go 1.14?

Reply:
go 1.13 has GO111MODULE=on by default AFAIK. Build fails unless you set GO111MODULE=off

Reply:
Then it should go into the library helper for build to set that (same place we set up go env) - hack/lib/util/golang.sh

Reply:
ok, updated